bgpd/bgp_damp.c

```diff
@@ -700,16 +700,16 @@ bgp_show_dampening_parameters (struct vty *vty, afi_t afi, safi_t safi)
 
   if (CHECK_FLAG (bgp->af_flags[afi][safi], BGP_CONFIG_DAMPENING))
     {
-      vty_out (vty, "Half-life time: %ld min%s",
-                    damp->half_life / 60, VTY_NEWLINE);
+      vty_out (vty, "Half-life time: %lld min%s",
+	       (long long )damp->half_life / 60, VTY_NEWLINE);
```

Comment:
I agree that the alignment for these vty_out's need to be fixed, but I see the surrounding code using spaces and not tabs to indent. Should probably use spaces here.